ecosystem/sep-0010.md

```diff
@@ -139,7 +139,81 @@ operation of the challenge is the **Home Domain**, and that the web auth domain
 GET <WEB_AUTH_ENDPOINT>
 ```
 
-Request Parameters:
+##### Request Header:
+
+Optionally, the server may forbid unauthorized calls to `GET <WEB_AUTH_ENDPOINT>`. This could be done to protect an
+endpoint against bad actors or limit access only to allowed applications.
+
+In that case, the client must add an `Authorization` header, and the server should respond to all requests with missing
+header with an error (see below). Server may choose to only accept calls from trusted applications, responding with an
+error to all other requests (see below).
+
+Authorized `WEB_AUTH_ENDPOINT` accepts `Authorization Bearer: <token>`, where token is [JSON Web Token](https://jwt.io/)
+authorizing the request.
+
+Client must specify the following claims:
+
+- `iat` (the time at which the JWT was issued
+  [RFC7519, Section 4.1.6](https://tools.ietf.org/html/rfc7519#section-4.1.6)) — current timestamp (for example
+  `1530644093`)
+- `exp` (the expiration time on or after which the JWT must not be accepted for processing,
+  [RFC7519, Section 4.1.4](https://tools.ietf.org/html/rfc7519#section-4.1.4))
+- `host` domain of the server to which the request is being made for.
+- `path` the path of the request (`<WEB_AUTH_ENDPOINT>`)
```

Comment:
I'm not sure it adds any value to split the host and path, since we really only wanted to exclude query parameters. I think we should continue using the full endpoint representation so we match the toml and `iss` field in the SEP-10 token.

So I think we have a couple options. We can do `path` (including the host), `web_auth_endpoint` to match the toml naming, or `aud` which seems like the [standardized](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3) name for what we're trying to do here. I think I prefer `aud` but I don't feel strongly.

Reply:
Agree on that

Reply:
I decided to use `web_auth_endpoint` instead as Java library uses set of strings in `aud`, and I think we should only allow single string there.